composer/utils/eval_client/mosaicml_lambda_eval_client.py

```diff
@@ -53,6 +53,9 @@ def invoke(self, payload: List[List[List[Dict[str, str]]]]) -> List[List[List[bo
                         log.error(f'Failed to get code eval output after {self.num_retries} retries. Error: {e}')
                     log.warning(f'Failed to get code eval output, retrying in {self.backoff**i} seconds.')
                     time.sleep(self.backoff**i)
+                elif e.status == 401:
```

Comment:
nit
```suggestion
                elif e.status == HTTPStatus.UNAUTHORIZED:
```
add `from http import HTTPStatus` above from the standard py library https://docs.python.org/3/library/http.html